<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerClientPane</class>
 <widget class="QWidget" name="ServerClientPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolBar" name="toolBar0">
       <property name="windowTitle">
        <string>edit</string>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <addaction name="actionBack"/>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolBar" name="toolBar">
       <property name="windowTitle">
        <string>edit</string>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <addaction name="actionRefresh"/>
       <addaction name="separator"/>
       <addaction name="actionListView"/>
       <addaction name="actionGridView"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="wProgressBar" name="widget" native="true"/>
   </item>
   <item>
    <widget class="ady::TreeView" name="treeView">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Refresh_16x.svg</normaloff>:/Resource/icons/Refresh_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Backwards_16x.svg</normaloff>:/Resource/icons/Backwards_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionGridView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/ShowGrid_16x.svg</normaloff>:/Resource/icons/ShowGrid_16x.svg</iconset>
   </property>
   <property name="text">
    <string>GridView</string>
   </property>
  </action>
  <action name="actionListView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/ListView_16x.svg</normaloff>:/Resource/icons/ListView_16x.svg</iconset>
   </property>
   <property name="text">
    <string>ListView</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Download_16x.svg</normaloff>:/Resource/icons/Download_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionPermissions">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/PageLock_16x.svg</normaloff>:/Resource/icons/PageLock_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Permissions</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/Cancel_16x.svg</normaloff>:/Resource/icons/Cancel_16x.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="icon">
    <iconset resource="../../r.qrc">
     <normaloff>:/Resource/icons/NewFileCollection_16x.svg</normaloff>:/Resource/icons/NewFileCollection_16x.svg</iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ady::TreeView</class>
   <extends>QTreeView</extends>
   <header>components/treeview.h</header>
  </customwidget>
  <customwidget>
   <class>wProgressBar</class>
   <extends>QWidget</extends>
   <header location="global">w_progress_bar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../r.qrc"/>
 </resources>
 <connections/>
</ui>
